server/plugin/command.go

```diff
@@ -353,7 +362,16 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		return "--exclude feature currently support on organization level."
 	}
 
-	if err := p.Subscribe(ctx, githubClient, args.UserId, owner, repo, args.ChannelId, features, flags); err != nil {
+	repoMatch, err := p.getExcludedRepo(args)
```

Comment:
Shouldn't we support more than one excluded repo here?

Reply:
@mickmister 1/5 we may add support for the same in next release. User may call the command multiple times if needed. Do you want me to add it in this PR?

Reply:
@sanjaydemansol Most of the code related to the exclude feature seems to imply it works with multiple repos. I need to review the original PR to understand the code better as I wasn't a reviewer on the PR.

> Shouldn't we support more than one repo here?

I said this with the assumption that the feature already supports it in general, and that I genuinely don't know if we need to support that in this piece of code as well